modules/nf-core/msisensorpro/pro/main.nf

```diff
@@ -0,0 +1,56 @@
+process MSISENSORPRO_PRO {
+    tag "$meta.id"
+    label 'process_low'
+
+    conda "${moduleDir}/environment.yml"
+    container "${ workflow.containerEngine == 'singularity' && !task.ext.singularity_pull_docker_container ?
+        'https://depot.galaxyproject.org/singularity/msisensor-pro%3A1.3.0--hfef96ef_0':
+        'quay.io/biocontainers/msisensor-pro:1.3.0--hfef96ef_0' }"
+
+    input:
+    tuple val(meta), path(cram)
+    path("${cram}.crai")
+    path(list)
+    path(fasta)
+    path("${fasta}.fai")
```

Comment:
```suggestion
    tuple val(meta), path(input), path(index)
    tuple val(meta2), path(list)
    tuple val(meta3), path(fasta)
    tuple val(meta4), path(fai)
```
It's better to have the input and it's index on the same line as these files are inherently linked and shouldn't be split up anyways. I also changed the name to `input` and `index` because not only CRAM files are allowed but also BAM files

I added some meta maps to the references to keep in line with the other nf-core modules